### PR TITLE
Move dt_start out of RRuleProperties

### PR DESCRIPTION
**Pull request by omid**

### CODE REVIEW

**Comment by omid** [comment]
@ralpha you recently disabled github checks on PRs. I didn't revert it back.
Even tests are less safe than before (testing all features vs every combination of them)

**Comment by omid** [comment]
@fmeringdal you recently disabled github checks on PRs. I didn't revert it back.
Even tests are less safe than before (testing all features vs every combination of them)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: rust
```
Comment (fmeringdal) [inline .travis.yml:1]: This file can be removed as Codecov action is not in `.github/workflows/codecode.yml`

File: rrule/src/parser/rrulestr.rs [hunk 2]
```diff
@@ -1078,12 +1060,40 @@ mod test {
         }
     }
 
+    #[test]
```
Comment (fmeringdal) [inline rrule/src/parser/rrulestr.rs:1063]: This test was removed in #52 , so it needs to be deleted.

**Review by fmeringdal — CHANGES_REQUESTED** [review]
Looks good to me! Just two things that should be deleted before we merge.
I will fix the GH Actions so that we can run tests as well.

**Comment by omid** [comment]
Thanks @fmeringdal

**Comment by fmeringdal** [comment]
Thanks for your patience @omid :) I will be more responsive from here on

**Pull request merged**